### PR TITLE
changed path handling to be OS agnostic

### PR DESCRIPTION
**Pull request by MarcoKe**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context / Related issue

Since Windows uses backslashes instead of forwardslashes, replacing parts of a path based on forwardslashes resulted in an error under Windows.

[https://github.com/facebookresearch/mbrl-lib/issues/152](https://github.com/facebookresearch/mbrl-lib/issues/152)

## How Has This Been Tested (if it applies)

The changes were manually tested using Windows 10 and a VM with Ubuntu 20.04. 

The test suite was executed using pytest and no new errors were found. 

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the [**CONTRIBUTING**](../../CONTRIBUTING.md) document and completed the CLA (see **CONTRIBUTING**).
- [x] All tests passed, and additional code has been covered with new tests. (Some tests already did not pass with a fresh clone of the main branch. Those still do not pass with the changes of course.

### CODE REVIEW

File: mbrl/diagnostics/training_browser.py [hunk 1]
```diff
@@ -50,10 +51,9 @@ def reloadExperiments(self, data):
         self.data = []
         for path in data:
             if path.endswith(SOURCE):
+                config_path = Path(path).parent / Path(".hydra/config.yaml")
```
Comment (luisenp) [inline mbrl/diagnostics/training_browser.py:54]: nit: I think you don't need to also wrap ".hydra/config.yaml" into a `Path` object.

**Review by luisenp — APPROVED** [review]
LGTM, thanks for reporting and adding a quick fix!

**Comment by luisenp** [comment]
Hi @MarcoKe. Can you rebase on top of the latest changes to `main`? I fixed some import issues that are causing CI to fail. After that, we are good to merge. Thanks!

**Comment by MarcoKe** [comment]
> Hi @MarcoKe. Can you rebase on top of the latest changes to `main`? I fixed some import issues that are causing CI to fail. After that, we are good to merge. Thanks!

Done. I also removed the unnecessary wrap in a Path object.

**Pull request merged**